cookbooks/zarr-stac-report/data-producers/index.qmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "Zarr + STAC for Data Producers"
+subtitle: "Approaches to structuring your data cubes using Zarr + STAC"
+---
+
+This section is targeted at people who are trying to disseminate data cubes. It addresses questions like:
+
+ - Should you use STAC?
+ - Should you use Zarr?
+ - Where should Zarr stores go in your STAC catalog?
+ - How much metadata should you pull up out of the Zarr dataset?
+ - How to pull metadata out of Zarr?
+
+## Is your data spatial-temporal?
```

Comment:
This content interesting but I think it is a safe assumption that people are here to disseminate geospatial data and we may just want to highlight some of the benefits of STAC in this section, like it's wide adoption, rich ecosystem and integrations, interoperability with tools like QGIS and extensibility.

Reply:
Yeah. I learned about a lot of different catalogs during the FROST conversations a few months ago, but I agree it's not the right spot to dump that.